packages/ui-components/src/components/QrCode/createQrCodeCanvas.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import { toDataURL } from 'qrcode';
+
+const CODE_SIZE = 320; // width/height at browser display size
+const CANVAS_SIZE = 395; // width/height at browser display size
+const DOWNLOAD_IMAGE_SCALE = 2; // render the canvas at double size to get a larger and clearer image download
+const SCALED_CODE_SIZE = CODE_SIZE * DOWNLOAD_IMAGE_SCALE;
+const SCALED_CANVAS_SIZE = CANVAS_SIZE * DOWNLOAD_IMAGE_SCALE;
```

Comment:
If any of these values are the same as the ones in admin panel, you could export them from a constants file inside this folder to reuse in admin panel, to save rewriting the same ones.